package.json

```
@@ -22,6 +22,7 @@
     "js-yaml": "3.14.0",
     "lodash": "4.17.20",
     "mocha-junit-reporter": "^2.0.0",
+    "npm": "^6.14.9",
```

I'd recommend pinning it down (that also goes for all the other dependencies, but you don't have to change those as part of your PR):

```suggestion
    "npm": "6.14.9",
```

---

Good idea! I did the same for other dependencies as well.